### ISSUE TITLE
ACER is still in use by L2 processing and should be restored to the meta-project (trunk) (Trac #1857)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1849]
The ACER project ($SVN/projects/AtmCscdEnergyReco) was retired 13 months ago. However it is still in use by L2 processing and should thus be restored to trunk so that tests are being run until we have a 1:1 replacement (unless that replacement already exists in which case filterscripts should be updated).

This currently blocks potential pass2 reprocessing.

Migrated from https://code.icecube.wisc.edu/ticket/1857
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:18", 
    "description": "The ACER project ($SVN/projects/AtmCscdEnergyReco) was retired 13 months ago. However it is still in use by L2 processing and should thus be restored to trunk so that tests are being run until we have a 1:1 replacement (unless that replacement already exists in which case filterscripts should be updated).\n\nThis currently blocks potential pass2 reprocessing.", 
    "reporter": "claudio.kopper", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067198646291", 
    "component": "combo reconstruction", 
    "summary": "ACER is still in use by L2 processing and should be restored to the meta-project (trunk)", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-09-12T14:54:04", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160912T17:10:30: olivas commented:

> ACER was included in IceRec V05-00-02 so processing should work just fine with that release.  Going forward, we can either bring ACER back (path of least resistance), or see if the Cascade group is OK with the alternative.  The latter isn't likely to happen before the collaboration meeting however, so if we need a new release of the trunk, resurrecting ACER is likely the way to go.  I'm not opposed to this.

**Comment by icecube-trac** [comment]
Trac update at 20160913T16:56:22: 
* olivas changed resolution from "" to "fixed"
* olivas changed status from "new" to "closed"
* olivas commented:

> In changeset 150004:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="150004"
> that was easier than expected.  ACER is back.  closes #1849
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:18: 
* olivas changed milestone from "September 2016" to ""
* olivas commented:

> Milestone September 2016 deleted
